csrankings-y.csv

```diff
@@ -759,8 +760,8 @@ Yuezhi Zhou,Tsinghua University,http://media.cs.tsinghua.edu.cn/~ns/eng/zyz/zhou
 Yufei Ding,Univ. of California - Santa Barbara,https://cs.ucsb.edu/~yufeiding,ZuNK2XkAAAAJ
 Yufei Tang,Florida Atlantic University,http://faculty.eng.fau.edu/tangy,tkzHKDoAAAAJ
 Yufei Tao 0001,Chinese University of Hong Kong,http://www.cse.cuhk.edu.hk/~taoyf,3FmJBHkAAAAJ
-Yufeng J. Tseng,National Taiwan University,http://ah.ntu.edu.tw/web/Teacher!one.action?tid=2147,NOSCHOLARPAGE
-Yufeng Jane Tseng,National Taiwan University,http://ah.ntu.edu.tw/web/Teacher!one.action?tid=2147,NOSCHOLARPAGE
+Yufeng J. Tseng,National Taiwan University,https://www.cmdm.tw/,prXvyPUAAAAJ

```

Comment:
Not obviously in CS. Please clarify.

Reply:
According to her [bio](https://www.cmdm.tw/index.html?content=janetseng), she is in the Department of Computer Science and Information Engineering.
[The website of the department](https://www.csie.ntu.edu.tw/en/member/Faculty) also shows that she is one of the Associate Chair of the department.